deploy/charts/cert-manager/templates/cainjector-rbac.yaml

```diff
@@ -67,11 +67,14 @@ metadata:
     helm.sh/chart: {{ include "cainjector.chart" . }}
 rules:
   # Used for leader election by the controller
-  # TODO: refine the permission to *just* the leader election configmap
+  # Done: refine the permission to *just* the leader election configmaps
```

Comment:
Can you remove this comment altogether? 😄

Reply:
Will do, btw do you want me to squash all the commits into a single one in the end?